tests/aws/services/apigateway/test_apigateway_basic.py

```diff
@@ -653,16 +651,13 @@ def test_malformed_response_apigw_invocation(self, create_lambda_function, aws_c
         lambda_resource = "/api/v1/{proxy+}"
         lambda_path = "/api/v1/hello/world"
 
-        create_lambda_function(
+        lambda_uri = create_lambda_function(
```

Comment:
nit: should be arn instead of URI (but not worth running the pipeline again because of this)